nailgun/entities.py

```diff
@@ -2864,8 +2864,13 @@ def path(self, which=None):
     def read(self, entity=None, attrs=None, ignore=None):
         """Fetch an attribute missing from the server's response.
 
+        Also add sync plan to the responce if needed, as
+        :meth:`nailgun.entity_mixins.EntityReadMixin.read` can't initialize
+        sync plan.
+
         For more information, see `Bugzilla #1237283
-        <https://bugzilla.redhat.com/show_bug.cgi?id=1237283>`_.
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237283>`_ and
+        `nailgun#261 <https://github.com/SatelliteQE/nailgun/issues/261>`_.
```

Comment:
Thanks for writing up #261, @abalakh. It's told me just what I needed to know to understand this PR.

I have no preference regarding whether to provide an inline explanation or a link to the issue. If an opinion is needed, then "let's keep the link," just because I don't want to burden anyone with more work.